templates/ruckus_fastiron_show_arp.template

```diff
@@ -8,11 +8,21 @@ Value STATUS (\S+)
 Value VLAN (\S+)
 
 Start
-  ^All\s+ARPs:\s+\d+,\s+maximum\s+capacity:\s+\d+
-  ^Total\s+(number\s+of\s+|)ARP\s+[Ee]ntries\s*:\s+\d+
-  ^Entries\s+in\s+default\s+routing\s+instance:
-  ^No\.\s+(IP\s+Address|IP)\s+(MAC\s+Address|MAC)\s+Type\s+Age\s+Port\s+Status(\s+VLAN|)
+  ^All\s+ARPs:\s+\d+,\s+maximum\s+capacity:\s+\d+ -> Layer2
+  ^Total\s+number\s+of\s+ARP\s+entries\s*:\s+\d+ -> Layer3
+  ^\s*$$
+  ^. -> Error 
+
+Layer2
+  ^No\.\s+(IP\s+Address|IP)\s+(MAC\s+Address|MAC)\s+Type\s+Age\s+Port\s+Status\s+VLAN

```

Comment:
Oops. I would have the transition these within **Start** to transition to the different states since these are the headers and we want to make sure we're keying off the correct data.

Reply:
Okay, I figured we should key off of the earliest thing we could determine which version of the OS we were on to eliminate checking lines we would never encounter. I will correct.

Reply:
Updated changes. My understanding is that once it changes state it doesn't go back unless we tell it to, which is why I moved the one line to the end of Layer2 state as it is after the data. Let me know if this is incorrect or there is a better way to handle a footer.